lib/ProductOpener/Products.pm

```diff
@@ -2954,6 +2955,32 @@ or "slack_CHANNEL_NAME" (B<warning> currently channel name is ignored, we post t
 
 =cut
 
+sub removeEmailValues ($product_ref) {
```

Comment:
In Product Opener (and most Perl programs I think), we use snake_case instead of camelCase for variable and function names.